### DISCUSSION TITLE
How can implement the choose image or use camera to take a picture on windows?

### DISCUSSION THREAD

**Discussion by taduyde** [microsoft/react-native-windows#10486] [Q&A] [answered]
As I use this plugin(https://github.com/react-native-image-picker/react-native-image-picker) to choose or take the picture on iOS or android. but on windows I didn't see any plugin, how can I implement that?

**Comment by chrisglein** [comment, marked as answer]
That one hasn't received an implementation for Windows yet. The only camera package that has Windows support is [react-native-camera](https://github.com/react-native-camera/react-native-camera) (searching the directory [here](https://reactnative.directory/?windows=true&search=camera), non-windows [here](https://reactnative.directory/?search=camera)). You could potentially harvest parts from react-native-camera to add Windows support to react-native-image-picker. 

General instructions here: https://microsoft.github.io/react-native-windows/docs/native-modules
